plugin/evm/atomic/vm/api.go

```diff
@@ -190,7 +190,7 @@ func (service *AvaxAPI) GetAtomicTxStatus(r *http.Request, args *api.JSONTxID, r
 		// Since chain state updates run asynchronously with VM block acceptance,
 		// avoid returning [Accepted] until the chain state reaches the block
 		// containing the atomic tx.
-		lastAccepted := service.vm.InnerVM.Blockchain().LastAcceptedBlock()
+		lastAccepted := service.vm.InnerVM.Ethereum().BlockChain().LastAcceptedBlock()
```

Comment:
`Ethereum().Blockchain()` is not consistently used everywhere - still use vm.Blockchain in 19 different places by my count. 

For example in `vm_test.go,` I think stuff like 

```
if ethBlk1Root := ethBlk1.Root(); !vm.blockChain.HasState(ethBlk1Root) {
		t.Fatalf("Expected blk1 state root to not yet be pruned after blk2 was accepted because of tip buffer")
	}
```
	
should be replaced `Ethereum().Blockchain().HasState()`

Reply:
They're part of extensible VM interface, the evm/vm has a direct access to these fields.